src/featherlight.js

```diff
@@ -619,7 +619,7 @@
 
 				this._$previouslyWithTabIndex.add(this._$previouslyTabbable).attr('tabindex', -1);
 
-				if (document.activeElement.blur) {
+				if (document.activeElement) {
```

Comment:
I can't easily check this, but #296 reports that some elements don't support the `blur` method? Maybe best to check both `document.activeElement && document.activeElement.blur`?

Reply:
@marcandre I'd have to try this on the corresponding project in IE11, but I think this also will result in a script breaking error when the element targeted is not available/null

Reply:
How would adding a check `&& document.activeElement.blur` break?